### ISSUE TITLE
Not supported on Angular 15

### ISSUE THREAD

**Issue by jpike88** [ng-select/ng-select#2164]
seems to work fine though. just a peer dependency warning

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: client@undefined
npm ERR! Found: @angular/common@15.0.0
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^15.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"<15.0.0" from @ng-select/ng-select@9.1.0
npm ERR! node_modules/@ng-select/ng-select
npm ERR!   @ng-select/ng-select@"^9.0.2" from the root project

```

**Comment by YanerTavuz** [comment]
https://github.com/ng-select/ng-select/pull/2163

**Comment by jpike88** [comment]
Works fine on 15 though, just need to convince the dependencies to update their package.json

**Comment by jpike88** [comment]
Or fork until they bother to do it

**Comment by arlopezg** [comment]
Fixed on #2169
